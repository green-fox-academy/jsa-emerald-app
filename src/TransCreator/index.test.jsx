import React from 'react';
import renderer from 'react-test-renderer';
import index from './index';

describe('<index/>', () => {
  it('render', () => {
    const tree = renderer.create(<index navigation={{}} />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
