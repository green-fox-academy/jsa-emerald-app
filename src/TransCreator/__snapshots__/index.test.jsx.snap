// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<index/> render 1`] = `
<index
  navigation={Object {}}
/>
`;
